### ISSUE TITLE
one of the variables needed for gradient computation has been modified by an inplace operation: [torch.cuda.FloatTensor [2, 5]], which is output 0 of ClampBackward, is at version 2;

### ISSUE THREAD

**Issue by xr-Yang** [cavalleria/cavaface.pytorch#10]
torch1.2 run 'train.sh',when loss.backforward. raise error like this:
one of the variables needed for gradient computation has been modified by an inplace operation: [torch.cuda.FloatTensor [2, 5]], which is output 0 of ClampBackward, is at version 2; expected version 0 instead. Hint: enable anomaly detection to find the operation that failed to compute its gradient, with torch.autograd.set_detect_anomaly(True)

and i use with torch.autograd.set_detect_anomaly(True): to locate the problem ,find that ''target_logit = cos_theta_1[torch.arange(0, embbedings.size(0)), label].view(-1, 1)'' raise error .

is the torch version problem? Thanks

**Comment by cavalleria** [comment]
Which loss are you using？please upgrade torch version to 1.4 and try it.

**Comment by xr-Yang** [comment]
> Which loss are you using？please upgrade torch version to 1.4 and try it.

Thanks for your reply. torch1.4 is OK.
I have another question.Why is the RGB_MEAN and RGB_STD parameter default set to 0.5, I think usually everyone sets it to [0.485, 0.456, 0.406], [0.229, 0.224, 0.225]

**Comment by cavalleria** [comment]
In most face recognition practices, mean and std is set to 0.5 to normalize input data.
